src/batch-client/index.ts

```diff
@@ -1529,6 +1557,17 @@ export class ZimbraBatchClient {
 			singleRequest: true
 		}).then(Boolean);
 
+	public suspendDeviceSync = (deviceId: String) =>
+		this.jsonRequest({
+			name: 'SuspendDevice',
+			namespace: Namespace.Sync,
+			body: {
+				device: {
+					id: deviceId
+				}
```

Comment:
can we create type for this as its getting reused many places

Reply:
It will help us if there will be more than one params, here its id only. lets keep this as it is.